docs/admin/install/microsoft_teams_setup.md

```diff
@@ -43,13 +38,18 @@
 3. Select the tile for the new Nautobot app. Click the blue “Add” button.
 4. Proceed to the [Install Guide](index.md#Install-Guide) section.
 
+## Nautobot Config
+
+- `microsoft_app_id` - This is the "Microsoft App ID" from Azure step 10.
+- `microsoft_app_password` - This is the "Microsoft App ID" from Azure step 13.
```

Comment:
Should this be `Client Secret` instead of `Microsoft App ID`?

```suggestion
- `microsoft_app_password` - This is the "Client Secret" from Azure step 13.
```